From 39f4030341db860aa1ce540ac1c537c2e6a9d938 Mon Sep 17 00:00:00 2001
From: Louis Lenders <xerox.xerox2000x@gmail.com>
Date: Thu, 6 Mar 2025 18:09:52 +0100
Subject: [PATCH 242/341] ntdll: Silence the noisy FIXME in
 RtlGetCurrentProcessorNumberEx.

This one is frequently flooding the console while testing various dotnet core programs.
---
 dlls/ntdll/rtl.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dlls/ntdll/rtl.c b/dlls/ntdll/rtl.c
index 643d6f3ce49..deee078faa2 100644
--- a/dlls/ntdll/rtl.c
+++ b/dlls/ntdll/rtl.c
@@ -842,7 +842,11 @@ BOOL WINAPI RtlSetCurrentTransaction(HANDLE new_transaction)
  */
 void WINAPI RtlGetCurrentProcessorNumberEx(PROCESSOR_NUMBER *processor)
 {
-    FIXME("(%p) :semi-stub\n", processor);
+    static int warn_once;
+
+    if (!warn_once++)
+        FIXME("(%p) :semi-stub\n", processor);
+
     processor->Group = 0;
     processor->Number = NtGetCurrentProcessorNumber();
     processor->Reserved = 0;
-- 
2.50.1

